src/pages/Donors/Donors.tsx

```diff
@@ -1,16 +1,58 @@
 import {
   IonButton,
   IonButtons,
+  IonContent,
   IonHeader,
+  IonItem,
+  IonLabel,
+  IonList,
   IonPage,
   IonTitle,
   IonToolbar,
 } from "@ionic/react";
+import {
+  CustomerConversationsListQuery,
+  useCustomerConversationsListQuery,
+} from "generated/graphql";
 import useAuth from "hooks/useAuth/useAuth";
 import React from "react";
 
+type ConversationItemType = CustomerConversationsListQuery["conversations"][0];
+
+interface MessagePreview {
+  message: ConversationItemType["messages"][0] | undefined;
+}
+const MessagePreview: React.FC<MessagePreview> = ({ message }) => {
+  if (message) {
+    const contentType = message.content.type;
+    if (contentType === "text") {
+      return <p>{message.content.text}</p>;
+    }
+
+    return <p>{`Sent ${contentType}`}</p>;
+  }
+  return null;
+};
+
+interface ConversationItemProps {
+  conversation: ConversationItemType;
```

Comment:
don't think it's a good idea to define a component's argument based on a Query's type. (If anything it would be the opposite).

you fetch data based on what you need to render, and not render based on what is being fetched.

(not critical)